### PR TITLE
add explicit chromium algorithm listing page

### LINKED ISSUE #117: ERR_SSL_VERSION_OR_CIPHER_MISMATCH on P256_KYBER90S512 for Chromium release 0.7
Visiting https://test.openquantumsafe.org:6007/ using https://github.com/open-quantum-safe/oqs-demos/releases/download/0.7.0/chromium-ubuntu-0.7.0.tgz gives a `ERR_SSL_VERSION_OR_CIPHER_MISMATCH` error. Same for https://test.openquantumsafe.org:6010/ with the assumption of using `P256_NTRU_HPS2048509`. Maybe the paragraph with "Therefore, only the following hybrid algorithms will work" could directly link to the right port with matching working (hybrid) algorithms?

### PR DESCRIPTION
**Pull request by baentsch**
Fixes #117

### CODE REVIEW

File: nginx/fulltest/build-ubuntu.sh [hunk 1]
```diff
@@ -7,10 +7,10 @@ export NGINX_PATH=/opt/nginx
 export NGINX_VERSION=1.16.1
 
 # define the OQS releases to use; unset to deploy main branch
-export LIBOQS_RELEASE=0.4.0
+export LIBOQS_RELEASE=0.7.1
 
 # defines the OQS-OpenSSL version to use; be sure to match with liboqs version
-export OPENSSL_RELEASE=OQS-OpenSSL_1_1_1-stable-snapshot-2020-08
+export OPENSSL_RELEASE=OQS-OpenSSL_1_1_1-stable
```
Comment (dstebila) [inline nginx/fulltest/build-ubuntu.sh:13]:
```suggestion
export OPENSSL_RELEASE=OQS-OpenSSL_1_1_1-stable-snapshot-2022-01
```

**Review by dstebila — APPROVED** [review]

File: nginx/fulltest/build-ubuntu.sh (hunk 1 above)
Comment (dstebila) [reply nginx/fulltest/build-ubuntu.sh:13]: Good catch, that was just a mistake on my part and not intentional. I rather wish I had picked shorter tag names when I started out, but alas that's what we're stuck with.

Comment (baentsch) [reply nginx/fulltest/build-ubuntu.sh]: So changed.

File: nginx/fulltest/build-ubuntu.sh [hunk 2]
```diff
@@ -7,10 +7,10 @@ export NGINX_PATH=/opt/nginx
 export NGINX_VERSION=1.16.1
```
[the remark this answers is not in the document]
Comment (baentsch) [reply nginx/fulltest/build-ubuntu.sh]: Update done. Deployed at https://test.openquantumsafe.org/ and all algorithms successfully tested against that deployment.

**Pull request merged**